lib/geoengineer/resources/aws/organizations/aws_organizations_organization.rb

```diff
@@ -0,0 +1,30 @@
+########################################################################
+# AwsOrganizationsOrganization is the +aws_organizations_organization+ terrform resource,
+#
+# {https://www.terraform.io/docs/providers/aws/r/organizations_organization.html Terraform Docs}
+########################################################################
+class GeoEngineer::Resources::AwsOrganizationsOrganization < GeoEngineer::Resource
+  after :initialize, -> {
+    _terraform_id -> { NullObject.maybe(remote_resource)._terraform_id }
+  }
+
+  # A fixed string is used here because an account can only have one
+  # organization, so if multiple get defined, we want geo to error.
+  after :initialize, -> { _geo_id -> { "aws_organization" } }
```

Comment:
Not that I think it's worthwhile to optimize for this at this time, but this seems to mean if someone were trying to use Geo the span multiple AWS accounts (which we could conceivably want to do in a distant future so that we can reference SGs across VPC-peered accounts—though this understandably changes the Geo model) we would not be able to do that unless said accounts were all under the same organization ID (which they likely will be for us).

For now I think this is a good limitation to introduce, as it's easy to fix should we choose to change.

Reply:
Yes, but I think in a multi-provider environment to manage multiple accounts in one pass, there'd be tons of these geo_id clashes in our own usage. I'd geo_id is used in conjunction with a provider name or something.

Also note this is not for organizational units, which is how you can do topologies within an org. Terraform doesn't have a resource for managing OUs, it seems.